web/src/components/storage/ProposalPage.jsx

```diff
@@ -232,6 +232,7 @@ export default function ProposalPage() {
       <ProposalSettingsSection
         availableDevices={state.availableDevices}
         encryptionMethods={state.encryptionMethods}
+        volumeTemplates={state.volumeTemplates}
```

Comment:
In a talk with @imobachgs, we concluded that this kind of errors could be avoid with the type checking. To do so, the  `volumeTemplates` prop should be required and not have a default value. The consumer is the responsible of giving the empty array if it's the case.

Reply:
Just a comment for bearing it in mind when activating the ts-check for the whole .jsx files.